assets/app/actions/routeActions.js

```diff
@@ -0,0 +1,12 @@
+import { navigationTypes } from '../constants';
+import store from '../store';
+
+export default {
+  redirect(path) {
+    store.dispatch({
+      type: navigationTypes.UPDATE_ROUTER,
```

Comment:
This is the type of action creator I'd like to see-- we do this in a few places and it's an easy thing to extract. I'm happy to run in and make this happen everywhere-- it's a quick quick quick refactor.